components/service/sync-logins/README.md

```diff
@@ -10,6 +10,16 @@ The **Firefox Sync - Logins Component** provides a way for Android applications
 
 ## Usage
 
+### Before using this component
+
+The `mozilla.appservices.logins` component collects telemetry using the [Glean SDK](https://mozilla.github.io/glean/).
+Applications that send telemetry via Glean *must ensure* they have received appropriate data-review following
+[the Firefox Data Collection process](https://wiki.mozilla.org/Firefox/Data_Collection) before integrating this component.
+
+Details on the metrics collected by the `mozilla.appservices.logins` component are available
+[here](https://github.com/mozilla/application-services/tree/master/docs/metrics/logins/metrics.md)
```

Comment:
This page is currently a 404: is this expected?

Reply:
Yes; it'll be live when the upstream logins-component PR lands.

Reply:
I just checked to confirm, and this link is now live as expected.